kernel/dynamic.c

```diff
@@ -43,9 +53,9 @@ static k_thread_stack_t *z_thread_stack_alloc_pool(size_t size)
 
 	size = Z_KERNEL_STACK_SIZE_ADJUST(size);
 
-	if (size > CONFIG_DYNAMIC_THREAD_STACK_SIZE) {
-		LOG_DBG("stack size %zu is > pool stack size %d", size,
-			CONFIG_DYNAMIC_THREAD_STACK_SIZE);
+	if (size > DYNAMIC_STACK_SIZE) {
```

Comment:
can't you do `size > Z_THREAD_STACK_SIZE_ADJUST(CONFIG_DYNAMIC_THREAD_STACK_SIZE)` ?